PumaGrasshopper/ComponentPumaBulkInput.cs

```diff
@@ -0,0 +1,144 @@
+﻿/**
+ * Puma - CityEngine Plugin for Rhinoceros
+ *
+ * See https://esri.github.io/cityengine/puma for documentation.
+ *
+ * Copyright (c) 2021-2023 Esri R&D Center Zurich
```

Comment:
Very minor but now it's already 2024 😄

Reply:
I think we need to change that for all the files, I will do a follow-up PR with only this change